LogMonitor/src/LogMonitor/pch.h

```diff
@@ -14,7 +14,6 @@
 #include <stdio.h>
 #include <array>
 #include <memory>
-#include <variant>

```

Comment:
what was `<variant>` being used for initially?

Reply:
removed this change, will include in next PR, this gives build warnings in cpp14 but doesn't seem to have any impact.